packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -2590,22 +2460,9 @@ const useEchartsCartesianConfig = (
     ]);
 
     const eChartsOptions = useMemo(() => {
-        // Apply category data to axes when using stacked bar data arrays (for border radius)
-        const categoryFieldHash = validCartesianConfig?.layout.flipAxes
-            ? validCartesianConfig?.layout?.yField?.[0]
-            : validCartesianConfig?.layout?.xField;
-
-        const modifiedAxes = applyStackedBarCategoryData(
-            axes,
-            stackedSeriesWithColorAssignments || [],
-            rows,
-            !!validCartesianConfig?.layout.flipAxes,
-            categoryFieldHash,
-        );
-
```

Comment:
reverted this change. this was a bad _hack_ and it's reverted to what was there before.